src/index.js

```diff
@@ -2,9 +2,29 @@ import './style.css';
 import RecycleImg from './recycle.svg';
 import MoreImg from './more.svg';
 import TrashImg from './delete.svg';
+import { drag, drop, allowDrop } from './drag_drop.js';
+import updateTasks from './status.js';
```

Comment:
- [ ] Loading any of the index.html shows a blank empty page. Why is this so? In this branch, you're expected to add interactiveness to the page not remove the previous work done. Kindly include the previous work done and ensure interactiveness is added to the todo list items.

Reply:
- [ ] In addition, kindly ensure you only have a single `index.html` page.

Reply:
Hi Mike. 
Thanks for pointing out those problems. I have already solved them.
To run the app now you have different choices. 

### Setup

```cmd
git clone  https://github.com/Metsanpeitto/To-do-list_m2_w2.git
cd ./To-do-list_m2_w2
```

### Install

```cmd
npm install
```

### Usage

```cmd
npm start
```


or 

### Run HTML file

```cmd
open the ./dist/index.html in your favorite browser 
```